Backend/ReleaseProcess.md

```diff
@@ -0,0 +1,50 @@
+Release process 📦
+==================================
+
+Our backend has a micro-service architecture. 
+To help improve the release process we are using [Semantic Release](https://semantic-release.gitbook.io/semantic-release/) in our commits.
+
+To automate finding the next version, we use [go-semrel-gitlab](https://juhani.gitlab.io/go-semrel-gitlab/) in our pipelines.
+
+
+## Merge requests
+
+Every merge request should run a pipeline to build, test and lint those changes, and only when everything is ✅ the merge can be done. 
+
+The stages for the merge requests are:
+
+![Merge request](./assets/merge_request.png)
+
+- `BuildMR` - Build the project
+- `LintMR` - Run lint checks
+- `TestsMR` - Run tests 
+
+
+## After merge
+
+After the merge, a new pipeline should run and effectively release the new build.
+
+The stages are: 
+
+![Merge request](./assets/merged.png)
```

Comment:
As we talked last days, we need to review this stages, here are missing stages for helm (lint, package) and swagger (lint, generate, publish)